diagnostic.md

```diff
@@ -7,25 +7,25 @@ Write your answers inside this file, where it's indicated by the comments.
     repository in your answer.
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    <!-- git clone <ssh> (in whatever folder you want the local repo to be in) git branch <new branch name to work in> -->
     ```
 
 2.  What do you do after cloning a repository, but before starting work?
 
-    <!-- Remove this comment and place your answer here. -->
+    <!-- create a new branch and then switch to it. Or is this where creating a remote comes in, becuase I think I'm still way off on understanding that. I thought a git clone <ssh> creates a remote for you? -->
```

Comment:
this is where you create a new branch, creating a remote is adding a different repository on github that you can pull from.